### ISSUE TITLE
Proble to load 432 the product page admin side, of a product with 432 combinations

### ISSUE THREAD

**Issue by Maofree** [PrestaShop/PrestaShop#17072]
hi
today a my custumer has created a product like normal but with many combinations for a total of 432, due from a mix of size and colors

it is necessary to wait more than 2-3 minutes to wait for the end of the loading of all the product data, only after this time all the combinations are displayed. I contacted the hosting to get more information to understand if it was a server or client side resource problem. He tells me that the problem is the javascript side in loading the data into the visitor's browser, in fact I believe that first all the data is loaded in the browser cache and then loaded in the list of combinations. I think the problem is that all those sections are actually only one form now, so it requires a lot more resources in loading all this data. Even in the rescue there are problems. I uninstalled the fastbay module which could further slow these saves.
I also did a test in incognito mode of chrome, but with the same loading problems.
in debug mode I don't see results, the bar is hidden.

the problem is present with the product with ID = 21052

I'll send you khouloudbelguith the access to this site to get more infos

bye

**Screenshots**
![3](https://user-images.githubusercontent.com/653230/71998746-7cb8c100-3240-11ea-8cc7-afbfdb5c75cb.png)
![4](https://user-images.githubusercontent.com/653230/71998748-7cb8c100-3240-11ea-98bb-9c855c94a881.png)

**Additional information**
PrestaShop version: 1.7.6.1
PHP version: 7.2

**Comment by sarahdib** [comment]
Duplicate of #9809
